### PR TITLE
Service snowflake

### PR DESCRIPTION
**Pull request by BatteryCandy**

### CODE REVIEW

File: products/snowflake/event_examples/query_history/authorization_add_permission.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  "SCHEMA_NAME": null,
+  "SECONDARY_ROLE_STATS": "{\"roleNames\":[],\"roleCount\":0,\"roleIds\":[]}",
+  "SESSION_ID": "47121867651652",
+  "START_TIME": "1716999270.649000",
+  "TOTAL_ELAPSED_TIME": "55",
+  "TRANSACTION_BLOCKED_TIME": "0",
+  "TRANSACTION_ID": "0",
+  "USER_NAME": "ALFRED_ADMIN",
+  "WAREHOUSE_ID": "1",
+  "WAREHOUSE_NAME": "GOTHSAM_WAREHOUSE",
```
Comment (dtocco) [inline products/snowflake/event_examples/query_history/authorization_add_permission.json:74]: typo with gotham

**Review by dtocco — APPROVED** [review]
🔥

File: products/snowflake/snowflake.product.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        name: About the Query History View
+        description: This Account Usage view can be used to query Snowflake query history by various dimensions (time range, session, user, warehouse, etc.) within the last 365 days (1 year).
+        url: https://docs.snowflake.com/en/sql-reference/account-usage/query_history
+  - id: sessions
+    name: Session History
+    references:
+      - id: about_sessions
+        name: About Sessions View
+        description: This Account Usage view provides information on the session, including information on the authentication method to Snowflake and the Snowflake login event. Snowflake returns one row for each session created over the last year.
+        url: https://docs.snowflake.com/en/sql-reference/account-usage/session
```
Comment (dodza-appomni) [inline products/snowflake/snowflake.product.yml:31]: Missing an "s" at the end of this URL

**Review by dodza-appomni — APPROVED** [review]
LGTM

**Pull request merged**